tests/Functional/app/config/services_test.yaml

```diff
@@ -0,0 +1,13 @@
+services:
+  _defaults:
+    autowire: true
+    autoconfigure: true
+
+  Silverback\ApiComponentBundle\Features\Bootstrap\:
+    resource: '../../../../features/bootstrap/*'
+
+  Silverback\ApiComponentBundle\Features\Bootstrap\DoctrineContext:
+    arguments:
```

Comment:
There is no need to inject those arguments, as autowiring injects them automatically. Please set DoctrineContext as following:
```php
public function __construct(ManagerRegistry $doctrine, JWTManagerInterface $jwtManager, IriConverterInterface $iriConverter)
```